### ISSUE TITLE
Make Swift.org/migration the landing page instead of a redirect

### ISSUE THREAD

**Issue by hborla** [apple/swift-migration-guide#54]
Currently, Swift.org/migration redirects to https://www.swift.org/migration/documentation/migrationguide/. We should fix this to make Swift.org/migration the landing page.

**Comment by mattmassicotte** [comment]
I think this is being done here: https://github.com/swiftlang/swift-org-website/pull/726

**Comment by hborla** [comment]
No, that PR is just updating the link on the website for when this issue is fixed. cc @shahmishal

**Comment by mattmassicotte** [comment]
ah, well at least related 😅

**Comment by shahmishal** [comment]
What should the landing page contain? I am assuming other migration guides and link to Swift 6 migration doc.

**Comment by hborla** [comment]
>What should the landing page contain? I am assuming other migration guides and link to Swift 6 migration doc.

I think the landing page should be this page: https://www.swift.org/migration/documentation/migrationguide/

I don't think the other older migration guides are super relevant anymore. We can evolve the landing page if/when there's a future language mode to document the migration for.
